src/main/java/org/jabref/gui/dialogs/BackupUIManager.java

```diff
@@ -88,12 +91,13 @@ private static Optional<ParserResult> showReviewBackupDialog(
                 List<DatabaseChange> changes = DatabaseChangeList.compareAndGetChanges(originalDatabase, backupDatabase, changeResolverFactory);
                 DatabaseChangesResolverDialog reviewBackupDialog = new DatabaseChangesResolverDialog(
                         changes,
-                        originalDatabase, "Review Backup"
+                        originalDatabase, "Review Backup",
+                        libraryTab
```

Comment:
Nono, there should be no save for restored backups. Please do not overwrite backups.